### PR TITLE
Media clock timer documentation updated

### LINKED ISSUE #127: Add 6.2 guide content
- [ ] App Services
- [ ] Block old head units
- [ ] Video streaming background string
- [ ] Screen Manager static icon
- [ ] Updated SetMediaClockTimer initializers (#120)
- [ ] OnSystemCapability

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Fixes #127 - media clock timer 

This pull request **adds new content**.

## Summary of Changes
* Updated all code samples to use the new media clock timer convenience inits.
* Moved the “Updating the Audio Indicator” info to the top of the page since the code samples use the audio indicator.

### CODE REVIEW

File: docs/Displaying a User Interface/Media Clock/index.md [hunk 1]
```diff
@@ -1,78 +1,50 @@
...
-
 ###### Generic HMI
 ![Media Clock Appearance](assets/generic_mediaclock.png)
 
 ###### Ford HMI
 ![Menu Appearance](assets/ford_mediaclock.png)
 
-## Counting Up
-In order to count up using the timer, the "bottom end" of the time will be 0:00, you can set the progress "start time", and the "top end". For example, if you are starting a song at `0:00` and it will end at `4:13`, you will end with a timer starting at `0:00`, ending at `0:00` with zero progress. This timer will automatically increment every second.
+## Updating the Audio Indicator
```
Comment (joeljfischer) [inline docs/Displaying a User Interface/Media Clock/index.md]: Can we move this section back to the end? Or to a different section of documentation? It feels odd putting it right under the introduction to the media clock, when we've just spent time explaining the media clock. I know that below code documentation mentions it, which is probably why you moved it up here. Also, are there any pictures using the `generic_hmi` we can use, or do they not implement it yet?

Perhaps under the count up section, you can add a `!!! NOTE` saying something like:

> The play / pause indicator parameter is used to update the play / pause button to your desired button type. This is explained below in the section "Updating the Audio Indicator"

File: docs/Displaying a User Interface/Media Clock/index.md [hunk 2]
```diff
@@ -1,78 +1,50 @@
...
-sdlManager.send(mediaClock)
-```
+For example, a radio app will probably want two button states: play and stop. A music app, in contrast, will probably want a play and pause button. If you don't send any audio indicator information, a play / pause button will be displayed.
 
-The following code will create a timer starting at `0:00`, ending at `4:13` and already at `2:20` of progress.
+## Counting Up
+In order to count up using the timer, you will need to set a start time that is less than the end time. The "bottom end" of the media clock will always start at `0:00` and the "top end" will be the end time you specified. The start time can be set to any position between 0 and the end time. For example, if you are starting a song at `0:30` and it ends at `4:13`, the media clock timer progress bar will start at the `0:30` position and start incrementing up automatically every second until it reaches `4:13`. The current position label will start counting upwards from `0:30` and the remaining time label will start counting down from `3.43`. When the end is reached, both labels will read `0:00` and the progress bar will stop moving.
```
Comment (joeljfischer) [inline docs/Displaying a User Interface/Media Clock/index.md]: * `3.43` -> `3:43`

> When the end is reached, both labels will read `0:00` and the progress bar will stop moving.
* Only the current time will state `4:13` and the remaining time label will state `0:00`

File: docs/Displaying a User Interface/Media Clock/index.md [hunk 3]
```diff
@@ -81,23 +53,23 @@ When pausing the timer, it will stop the timer as soon as the request is receive
 
 ##### Objective-C
 ```objc
-SDLSetMediaClockTimer *mediaClock = [[SDLSetMediaClockTimer alloc] initWithUpdateMode:SDLUpdateModePause];
+SDLSetMediaClockTimer *mediaClock = [SDLSetMediaClockTimer pauseWithPlayPauseIndicator:SDLAudioStreamingIndicatorPlay];
```
Comment (joeljfischer) [inline docs/Displaying a User Interface/Media Clock/index.md]: We should probably add an example of pausing and altering the timer times.

File: docs/Displaying a User Interface/Media Clock/index.md (hunk 1 above)
Comment (NicoleYarroch) [reply docs/Displaying a User Interface/Media Clock/index.md]: 1. "Updating the Audio Indicator " section is now back at bottom.
2. Added new info about the play/pause indicator
3. Updated the screenshots so they match

File: docs/Displaying a User Interface/Media Clock/index.md (hunk 3 above)
Comment (NicoleYarroch) [reply docs/Displaying a User Interface/Media Clock/index.md]: Fixed. Added

File: docs/Displaying a User Interface/Media Clock/index.md (hunk 2 above)
Comment (NicoleYarroch) [reply docs/Displaying a User Interface/Media Clock/index.md]: Fixed both

**Pull request merged**